main.nf

```diff
@@ -167,46 +486,759 @@ process fastqc {
 
     script:
     """
-    fastqc -q $reads
+    fastqc -t "${task.cpus}" -q $reads
```

Comment:
Huh, I wonder if we should be doing this for all pipelines... 🤔

Reply:
Could you please add a PR to the main template for this?

Reply:
it's already in the template 😁

Reply:
Hah, great!